src/EnergyPlus/UnitarySystem.cc

```diff
@@ -12853,7 +12853,7 @@ namespace UnitarySystems {
                 if (this->m_FanOpMode == DataHVACGlobals::CycFanCycCoil) {
                     DataAirLoop::AirLoopFlow(AirLoopNum).FanPLR = this->FanPartLoadRatio;
                 } else {
-                    DataAirLoop::AirLoopFlow(AirLoopNum).FanPLR = 0.0;
+                    DataAirLoop::AirLoopFlow(AirLoopNum).FanPLR = 1.0;
```

Comment:
Also needs to default to 1 in DataAirLoop.hh

```
        // Default Constructor
        AirLoopFlowData()
            : DesSupply(0.0), DesReturnFrac(1.0), SysToZoneDesFlowRatio(0.0), ReqSupplyFrac(1.0), MinOutAir(0.0), MaxOutAir(0.0), OAMinFrac(0.0),
              Previous(0.0), SupFlow(0.0), ZoneRetFlow(0.0), ZoneRetFlowRatio(1.0), SysRetFlow(0.0), RecircFlow(0.0), LeakFlow(0.0),
              ExcessZoneExhFlow(0.0), FanPLR(0.0), OAFrac(0.0), OAFlow(0.0), FlowError(false), BypassMassFlow(0.0)
        {
        }
```

Reply:
Done